src/constants.js

```diff
@@ -207,3 +207,11 @@ export const SIDEBAR_VIEW_SKILLS: 'skills' = 'skills';
 export const SIDEBAR_VIEW_DETAILS: 'details' = 'details';
 export const SIDEBAR_VIEW_METADATA: 'metadata' = 'metadata';
 export const SIDEBAR_VIEW_ACTIVITY: 'activity' = 'activity';
+
+/* ------------------ HTTP Requests  ---------------------- */
+export const HTTP_GET = 'get';
```

Comment:
flow is fine with this?
I thought you had to do `export const HTTP_GET: 'get' = 'get';`
So that typeof is the first 'get'

Maybe they fixed it

Reply:
Nah, you have to do it that way. Right now its just any string.
https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzADEB5EsAXjAHIo45qBuTHAsAIQEEAlSmgIwCGAJybp8ATwAOAU2JkuvKpNlwo88gB8wKmWo49mLPITpwhwgFwbFfavBhjUQA

Reply:
interesting